### ISSUE TITLE
Cannot activate the device

### ISSUE THREAD

**Issue by Amitabitbul** [chiptuneXT/DoulCi-server2#3]
Hi,
everything working just fine, i mean, there is no errors in the php side.
but the itunes alert that the response details is not valid, are you sure the hardcoded private keys base64 are valid?
or it's something else?
i tried to play with the serials ( cert_user & cert_pass as described in the code )

im trying to unlock my device.

i was testing this service with itunes12, maybe version 12 has a protection from this?

Thanks!

**Comment by chiptuneXT** [comment]
You never unlock device with this server, because it have self-sign certs, iPhone check sign with built-in open key. You can fix it, if you put certs from _server_ to Lockdown folder and replace it.

As I say, it only for entertainment purposes. Enjoy!

**Comment by armenh7** [comment]
^^^ How would I be able to do that?

**Comment by ZetallicA** [comment]
I believe you have to pick up the certs from your server (after decrypting using fiddler's text wizard) and then reconstruct the PLIST file in lockdownd folder using the info. never tried this. there videos on youtube showing this. I need a little guidance in doing this.  zetallica@zetallica.com

**Comment by dev0zero** [comment]
can you explain about what certs are you tallking about. in certs/iphoneCA ?

**Comment by ZetallicA** [comment]
I don't remember anything about this :) 
I believe there is a private cert in the iDevice that needs to be signed (I dont even remmeber how signing certs works) 
The mechanism in the PHP file load the certs and the information from the iDevice, then using the signed cert info that includes the idevice info (which change 'unactivated' to Activated in Plist file) then send it back to the idevice either via itunes or via Wifi . 
this manipulation can be done in fiddler. 

All of this is just a GUESS , I dont know anything. 
I would love to work this with you. I have a bunch of iphones we can play with.  

email me your phone number/skype to  zetallica@zetallica.com . we will talk 

---

I heard someone found a trick to convince Lockdownd to activate. He is building an app . 

Lockdownd protocol  ==>   https://www.theiphonewiki.com/wiki/Usbmux  
http://iphonedevwiki.net/index.php/Lockdownd

http://www.libimobiledevice.org/docs/mobilesync.html

umm this is new ==> https://gist.github.com/copumpkin/149443

**Comment by maxandmax** [comment]
Hi again you can find me at skype by jcmax2010

**Comment by chiptuneXT** [comment]
Отправлено с iPhone

**Comment by chiptuneXT** [comment]
Hello everyone. This server has few errors that not allow activate device. Btw, if fix - you still get fail. Why? Need closed key for genuine certs from device. Closed keys located in the Albert server and on the some peoples (include me). Bruteforce like crazy. Hard like break Verisign or Symantec CA.
This server with fake certs and fake closed keys. It allow execute code without errors messages. Only device can be verify activation tickets. 
Best wishes, Michael.

**Comment by dev0zero** [comment]
it means script not working & it dead ? Question why iOS 8.3 -8.4 version stop working with iTunes 11?

**Comment by chiptuneXT** [comment]
No, script have few problems with algorithm (made my doulci). These problems can be fixable, but Apple changed algorithm again on the new iOS and iTunes builds. I not want burn potential exploit.
Sooner I will fix problems with iTunes 12. Apple require always have up-to date software, because not allow downgrades on the iOS :)

**Comment by dev0zero** [comment]
So for now bypassing iCloud is possible only if some one break root file system and publish Root Key here https://www.theiphonewiki.com/wiki/Firmware_Keys ???

**Comment by chiptuneXT** [comment]
Pff, I can post public key, if you want, Btw need private (closed) key.
Based on these server (yes, DoulCi-server2) I made this bypass solution:
https://twitter.com/chiptunext/status/583998074592301056
Get private key - too difficult.

**Comment by Mendim** [comment]
Is any way to get the private keys.

**Comment by maxandmax** [comment]
What you can say about this video https://www.youtube.com/watch?v=EXSnsmzCPBQ . Here is looks like iOS 8.4 and iTunes 12.0.1 and it works with old Dulci script but when i download this script is contain php script errors wtf?

**Comment by maxandmax** [comment]
some one can advice some instruments to decrypt root file system???

**Comment by themidnightgospel** [comment]
It says : "The Iphone "xx" could not be activated because the activation information was invalid. Please try again later"
